Makefile

```diff
@@ -41,3 +41,60 @@ javascript/dist/main.css: javascript/dist/main.js
 
 javascript/dist/main.js: javascript/package.json $(javascript_source)
 	cd javascript && npm install-test && npm run build
+
+RELEASED_VERSION = $(shell changelog latest)
+JAVA_VERSION = $(shell cd java && make version)
+RUBY_VERSION = $(shell cd ruby && make version)
+JAVASCRIPT_VERSION = $(shell cd javascript && make version)
+CURRENT_BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+
+language-versions-check:
+	@if [[ ($(JAVA_VERSION) != $(RUBY_VERSION)) || ($(JAVA_VERSION) != $(JAVASCRIPT_VERSION)) ]];\
+	then \
+	echo "Language are inconsistent!"; \
+	echo "Java: \t\t$(JAVA_VERSION)"; \
+	echo "JavaScript: \t$(JAVASCRIPT_VERSION)"; \
+	echo "Ruby: \t\t$(RUBY_VERSION)"; \
+	exit 1; \
+	fi
+.PHONY: language-versions-check
+
+release-version-check:
+	@if [[ "$(RELEASED_VERSION)" = "$(JAVA_VERSION)" ]]; \
```

Comment:
I lkmdx the pattern used above in `language-versions-check`. You can use it here too.

Reply:
Done in 08812fe1ad921c66abda889d9451b36b395525e4